### PR TITLE
Feature of version check. 

### PR DESCRIPTION
**Pull request by doomsdayer**
When reading a csb file, if the engine's reader version dismatch the version saved in the csb file, an assert will be triggered. User must go to a website to get a reader to match the csb file.

Removed the version compatible control code before.

### CODE REVIEW

**Comment by minggo** [comment]
Please fix conflict.

**Comment by doomsdayer** [comment]
The conflict has been resolved.

File: cocos/editor-support/cocostudio/ActionTimeline/CSLoader.h [hunk 1]
```diff
@@ -104,7 +104,6 @@ class CC_STUDIO_DLL CSLoader
     
     cocos2d::Node* createNodeWithFlatBuffersForSimulator(const std::string& filename);
     cocos2d::Node* nodeWithFlatBuffersForSimulator(const flatbuffers::NodeTree* nodetree);
-    std::string getCsdVersion() { return _csdVersion; }
```
Comment (minggo) [inline cocos/editor-support/cocostudio/ActionTimeline/CSLoader.h:107]: Break compatibility?

Comment (doomsdayer) [inline cocos/editor-support/cocostudio/ActionTimeline/CSLoader.h:107]: This function will not be used anymore~

**Pull request merged**